README.md

```diff
@@ -2,4 +2,18 @@ Alexandria is an open source language learning app that allows users learn langu
 
 Read about how we made it [here](https://alexandria-reader.github.io/).
 
-Alexandria is accessible [here](https://tryalexandria.com/).
\ No newline at end of file
+Alexandria is accessible [here](https://tryalexandria.com/).
+
+## Features Highlight
+
+### Light and dark modes
+
+![](../frontend/src/assets/light-dark.gif)
+
+### Add translation for selected words
+
+![](../frontend/src/assets/set-translation.gif)
```

Comment:
Looks great! At the end of this gif you can see you clicking on the controls to stop the recording, is that intentional?